lib/CMakeLists.txt

```diff
@@ -191,3 +191,5 @@ endif()
 #             with gcc 10.x and C++11/14, so we need to publish the
 #             C++17 standard
 target_compile_features(ocpp PUBLIC cxx_std_17)
+append_coverage_compiler_flags_to_target(ocpp)
```

Comment:
you can do this in `tests/CMakeLists.txt` so these flags don't get added even if you don't want to have coverage enabled